Networking/NetworkingTests/Remote/ProductsRemoteTests.swift

```diff
@@ -270,6 +298,7 @@ private extension ProductsRemoteTests {
                        downloads: [],
                        downloadLimit: -1,
                        downloadExpiry: -1,
+                       buttonText: "",
```

Comment:
Also here, if it's not present maybe it's better to set it as `nil`.

Reply:
(same as the above)